app/views/dashboard/service_calendars/_view_full_calendar.html.haml

```diff
@@ -89,15 +89,15 @@
                 %h4.panel-title
                   = t(:dashboard)[:service_calendars][:consolidated_request][:services][:header]
               .panel-body
-                - if @protocol.has_per_patient_per_visit? @service_request, @portal
+                - if @protocol.has_line_items_of_type? @service_request, @portal
```

Comment:
Should be called with parentheses as per the rails style guide: @protocol.has_line_items_of_type(@service_request, @portal)